examples/tensorflow/requirements.txt

```diff
@@ -1,6 +1,6 @@
 absl-py==0.10
 tensorflow_datasets==4.2.0
 tensorflow_hub
-tensorflow_addons==0.12.0
+tensorflow_addons==0.12.1
```

Comment:
We have double of tensorflow_addons in [tests/tensorflow/requirements.txt](https://github.com/openvinotoolkit/nncf/blob/develop/tests/tensorflow/requirements.txt), need change it too.

Reply:
Done